lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -137,20 +132,31 @@ export abstract class ClientApplication {
         };
     }
 
+    /**
+     * Generates a request with the default scopes.
+     * @param authRequest 
+     */
+    protected initializeRequestScopes(authRequest: BaseAuthRequest): BaseAuthRequest {
+        const request: BaseAuthRequest = { ...authRequest };
+        if (!request.scopes) {
```

Comment:
Should you also be checking for array length?

Reply:
resolved offline, don't need to check for length.